config/toml.go

```diff
@@ -280,6 +280,11 @@ enabled = {{ .GRPC.VersionService.Enabled }}
 [grpc.block_service]
 enabled = {{ .GRPC.BlockService.Enabled }}
 
+# The gRPC block results service returns block results for a given height. If no height
+# is given, it will return the block results from the latest height.
```

Comment:
This text might be before your separation of the services into `GetLatestHeight` and `GetHeight`. Atm in the code if the req.Height is 0 we do not return the latest height right?